utils/progressbar/progressbarmng.go

```diff
@@ -83,6 +85,15 @@ func (bm *ProgressBarMng) NewTasksWithHeadlineProg(totalTasks int64, headline st
 	return &prog
 }
 
+func (bm *ProgressBarMng) NewHeadLineDoubleProgBar(headLine string, counterLine string, counterLine2 string, color Color, totalTasks1 int64, optionalTotal1, optionalDone1, totalTasks2, DoneTasks2 *int64) *TasksWithHeadlineProg {
```

Comment:
Let's add a cealr descriotuo for this function, and also for all the other functions generating the various progress bars.

NewHeadLineDoubleProgBar --> NewHeadLineDoubleValProgressBar